plugins/inputs/logfile/logfile.go

```diff
@@ -304,18 +304,20 @@ func (t *LogFile) getTargetFiles(fileconfig *FileConfig) ([]string, error) {
 			continue
 		}
 
-		// Add another file blacklist here
 		fileBaseName := filepath.Base(matchedFileName)
 		if blacklistP != nil && blacklistP.MatchString(fileBaseName) {
 			continue
 		}
 		if !fileconfig.PublishMultiLogs {
+			t.Log.Debugf("Single file mode - current target: %s (ModTime: %v), candidate: %s (ModTime: %v)",
```

Comment:
nit: I think this can become excessive logging as well depending on the number of files that match.